src/services/message-formating/message-formatting.service.ts

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash"
+
+export class MessageFormattingService {
+  private static _instance: MessageFormattingService
+
+  public static getInstance(): MessageFormattingService {
+    if (_.isNil(MessageFormattingService._instance)) {
+      MessageFormattingService._instance = new MessageFormattingService()
+    }
+    return MessageFormattingService._instance
+  }
+
+  public format(formatToApply: string, text: string) {
+    return this[formatToApply](text)
+  }
+
+  public bold(text: string) {
```

Comment:
The return type is missing on these methods.
Also I suggest you to not wait too much time before adding a linter to improve your code and avoir mistakes.
I am a linter but I am not good enough to spot everything :p

Reply:
Oh! Ok! Will do that in another Pull Request!

Thanks for suggesting!